Completion/X/Command/_sublimetext

```diff
@@ -7,7 +7,16 @@ _arguments -s -S : \
   '(: -)'{-h,--help}'[display help information]' \
   '(-a -n --add --new-window)'{-n,--new-window}'[open new window]' \
   '--project[load specified project]:project file:_files' \
-  '(-s --stay)'{-s,--stay}'[keep application activated]' \
```

Comment:
[Sublime's docs][1] contain "The available flags will vary per operating system," so I'm hesitant to remove any flags without an ok from @okdana (who added the flag).

[1]: https://www.sublimetext.com/docs/command_line.html

Reply:
Would've been nice to also conditionally output flag - but at least Sublime does not seem to do anything either way on Ubuntu.

I can revert since you hesitate.

Reply:
Conditional behaviour is possible. What would the condition be?

Reply:
All arguments must exist in `subl --help` (by means of grepping??), otherwise not offered

I know it's not a panacea convention, but a lot of respectful utils offer their arguments on either `-h`/`--help` - Sublime does too

Reply:
`-s`/`--stay` is still there on macOS at least in `subl` 4113. If that option is platform-specific, like Daniel said, we could easily make it conditional. Determining the applicability of options by looking at `--help` output is possible, but not common; in this case  i think it'd be fine to just look at `$OSTYPE`

Reply:
> in this case i think it'd be fine to just look at `$OSTYPE`

"As far as you know" it's based on `$OSTYPE`. Maybe it's only on MacOS? Or it's only not on Linux? 😕
Blackbox approach (e.g. grepping) makes less assumptions on things we don't know for sure.

However, I am not going to tackle it anyway - especially now that you verified it's still a thing.

Reply:
> Maybe it's only on MacOS? Or it's only not on Linux?

Either/both of those conditions could be handled by checking `$OSTYPE`, and it's how most other functions with platform-specific options do it. But yeah i haven't actually checked to see which options are available on which platforms

> However, I am not going to tackle it anyway

That's fine, we can revisit later if we want